docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Live Context
+description: Powered by our Workstream Pattern Engine, Live Context enables the world’s first Temporally Grounded Copilot that understands your unique workflow
+---
+
+import {MiniSpacer} from "/src/components/Spacers";
+import Video from "/src/components/Video";
+
+# Applying Live Context to your Copilot
+
+:::info
+This feature is currently in Alpha and is now generally available to all users. It is only available to macOS & Windows users, with Linux support coming ASAP 🙏🏼.
```

Comment:
Currently in Beta**

Reply:
@tsavo-at-pieces we currently have this in the app as "Alpha", but can adjust if needed in a patch. Lmk thoughts.

Reply:
Resolution: Adjusting in-app to be Beta

Reply:
Resolved